File: Java/1602.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class Main {
    /* 절댓값 구하는 함수 */
    static double ABS(double pnum) {
        if( pnum >= 0 )
            return pnum;
        else
            return -pnum;
    }

    /* 메인 함수 */
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        DecimalFormat df = new DecimalFormat("#.##########");

        double num = scanner.nextDouble();

        System.out.println(
                df.format(Main.ABS(num))
        );
    }
}

File: C/1602.c
#include <stdio.h>
#include <malloc.h>

double ABS(double* pNum)
{
	if(*pNum >= 0)
		return *pNum;
	else
		return -*pNum;
}

int main(void)
{
	// 입력 및 초기화 
	double* num = (double*)malloc(sizeof(double));
	scanf("%lf", num);
	
	// 출력부 
	printf("%.10g", ABS(num));
	
	return 0;
}
